source/localizable/guides/app/design/bootstrap.en.html.markdown.haml

```diff
@@ -9,13 +9,7 @@ title: Add Bootstrap
 
 %p The app doesn't look very nice yet. Let's do something about that. We'll use the Twitter Bootstrap project to give us nicer styling really easily.
 
-Open the `Gemfile` in your text editor and below the line
-
-= preserve do
-  - code("ruby") do
-    source 'https://rubygems.org'
-
-add
+Open the `Gemfile` in your text editor and add the following line to the list of gems:
```

Comment:
@inbalg in this case they can just add this like to the first line of the file.. which is not good..
Do you have an idea of how to keep those "After the line" but make them not look the same like codes?